source/develop/release-management/features/storage/gluster-multiple-bricks-per-storage.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Support for multiple GlusterFS bricks on a block device(s)
+category: feature
+authors: dchaplyg
+feature_name: Support for multiple GlusterFS bricks on a block device(s)
+feature_modules: engine/vdsm
+feature_status: Planning
+---
+
+# Support for multiple GlusterFS bricks on a block device(s)
+
+# Summary
+
+This feature introduces ability to create multiple GlusterFS bricks on the same block device(s).
+
+In case of shared block device, engine should create a LVM volume group on top of it and use LVM logical volumes as a brick backend devices. That approach will allow management of the local storage alongside with support for thin provisioning for the GlusterFS and SSD caching.
```

Comment:
It's not clear what a 'shared block device' is. Shared by whom?

Reply:
You are right, it sounds weird here. The whole idea was to allow several bricks to share one (or more) physical block devices.